src/entities/person/person.ts

```diff
@@ -193,6 +193,24 @@ export interface IPersonAddresses {
   toJson: Function
   create: Function
 }
+export enum MessageSubscriptionActionTypesEnum {
+  OPTIN = 'optin',
+  OPTOUT = 'optout',
+  WITHDRAW = 'withdraw',
+}
+
+export type IMessageSubscriptionActionType = MessageSubscriptionActionTypesEnum.OPTIN | MessageSubscriptionActionTypesEnum.OPTOUT | MessageSubscriptionActionTypesEnum.WITHDRAW
```

Comment:
A small hack that you could use here for better extensibility:

`export type IMessageSubscriptionActionType = `${MessageSubscriptionActionTypesEnum}``